x-pack/plugins/apm/public/components/app/Settings/schema/schema_overview.tsx

```diff
@@ -139,17 +138,17 @@ export function SchemaOverview({
         <EuiFlexItem />
         <EuiFlexItem style={{ minWidth: '250px' }}>
           <EuiCard
-            icon={<EuiIcon size="xxl" type="documents" />}
+            icon={<EuiIcon size="xxl" type="node" />}
             title={i18n.translate(
               'xpack.apm.settings.schema.migrate.classicIndices.title',
-              { defaultMessage: 'Classic APM indices' }
+              { defaultMessage: 'APM Server binary' }
             )}
             display="subdued"
             description={i18n.translate(
               'xpack.apm.settings.schema.migrate.classicIndices.description',
               {
                 defaultMessage:
-                  'You are currently using classic APM indices for your data. This data schema is going away and is being replaced by data streams in Elastic Stack version 8.0.',
+                  'You are currently using APM server binary. This legacy option is deprecated since version 7.16 and is being replaced by a manages APM Server in Elastic Agent from version 8.0.',
```

Comment:
```suggestion
                  'You are currently using APM Server binary. This legacy option is deprecated since version 7.16 and is being replaced by a manages APM Server in Elastic Agent from version 8.0.',
```

Missed a capitalization